mkosi

```diff
@@ -1202,13 +1202,14 @@ def umount(where: str) -> None:
     run(["umount", "--recursive", "-n", where], stdout=DEVNULL, stderr=DEVNULL)
 
 
-@completestep('Setting up basic OS tree')
-def prepare_tree(args: CommandLineArguments, workspace: str, do_run_build_script: bool, cached: bool) -> None:
+@completestep('Setting up OS tree root')
+def prepare_tree_root(args: CommandLineArguments, workspace: str) -> None:
     if args.output_format == OutputFormat.subvolume:
         btrfs_subvol_create(os.path.join(workspace, "root"))
-    else:
-        mkdir_last(os.path.join(workspace, "root"), 0o755)
```

Comment:
Why remove this? Haven't tested this yet but I think we'll start getting failures if workspace/root doesn't exist when not using a subvolume no?

Reply:
Nvm, `mount_bind` makes sure the directory exists.